### PR TITLE
Cargonian Independence (QM is a Head)

### PR DESCRIPTION
**Pull request by ike709**
:cl: ike709
tweak: The Quartermaster is now a head of staff. Yes, really. They have bridge access and everything. Glory to Cargonia.
tweak: The HoP is no longer in charge of cargo, and has lost cargo access.
/:cl:

Pretty sure I changed everything relevant...

Part ? of the "Internal Affairs / Lawyer / HR" rework.

Cargo has pretty much always been its own thing. The HoP rarely exerts their authority of cargo, and the QM has been a Head in all but name for a long time.

### CODE REVIEW

**Comment by Qustinnus** [comment]
HoP is head of supply please take your low quality lore somewhere else like the dumpster :^)

**Comment by ike709** [comment]
\>lore

floyd pls

File: code/modules/jobs/job_types/captain.dm [hunk 1]
```diff
@@ -68,21 +68,21 @@ Head of Personnel
 	minimal_player_age = 10
 	exp_requirements = 180
 	exp_type = EXP_TYPE_CREW
-	exp_type_department = EXP_TYPE_SUPPLY
+	exp_type_department = EXP_TYPE_SERVICE
```
Comment (FlattestGuitar) [inline code/modules/jobs/job_types/captain.dm]: tfw only clown mains can play HoP now

**Comment by FlattestGuitar** [comment]
From a code standpoint it's *okay*.

From a design standpoint **why**

**Comment by Ziiro** [comment]
QM fills this really weird spot where they're at the top of an entire section of the station, but they're not actually a head? I understand that this change takes some of the job away from the HoP, but I think the QM not having head authority is a bit inconsistent.

**Comment by FlattestGuitar** [comment]
The problem with the QM is that they don't have nearly enough to do to warrant them being a Head.

Maybe if we added things for them to actually handle other than pressing a button we could make this somehow meaningful, but for right now I just don't see the point of this tweak.

File: code/modules/jobs/job_types/captain.dm (hunk 1 above)
Comment (ike709) [reply code/modules/jobs/job_types/captain.dm]: you got it bby

**Comment by ike709** [comment]
The QM currently can do as much as they want to do. 

Want to sit in your office and do nothing all day? You can do that. Want to be a cargo tech but with a cloak and stamp? You can do that.

Want to actually consider requests, handle resource distribution, manage your point-collecting ~monkeys~ minions, ensure every department is supplied, and respond to emergencies by providing helpful gear? You can do that too.

I'm trying to encourage the latter by giving the QM more autonomy and responsibility. The QM has as much to do as they want to have to do, currently.

File: code/modules/jobs/job_types/captain.dm (hunk 1 above)
Comment (ike709) [reply code/modules/jobs/job_types/captain.dm]: although I do like the idea of only clown mains being allowed to be HoP

**Comment by scrubmcnoob** [comment]
HOP already has enough to deal with in their line.(they're not even there half the time)
Realistically, has the HOP ever bothered to actually check in on cargo? Maybe in a blue moon but why not offload this cargo to the QM. 
HOP also needs to oversee the church, botany, kitchen, booze, theatrical department, and anything else the other heads don't have control over.
Realistically, a good HOP can't check in on all those as well as all of cargo and mining and still run his line for a good portion of the shift in an effective manner.

**Comment by ike709** [comment]
image

Democracy has spoken.

**Comment by Cactus2388** [comment]
Head of Personnel - distribute access, secondary captain

Chief Engineer - Maintain station

Chief Medical Officer - Over see medbay

Research Director - Ensure RnD gets done. Over view scientists/AI

Head of Security - Make sure security follows SoP and Spacelaw, fill in for security roles

Captain - Ensure heads of staff aren't out of line/fill in for missing ones

Quartermaster - mail crates, order crates, cargotide, "haha cargonia amirite fellas"

**Comment by Qustinnus** [comment]
HoP is not secondary captain as main role, hes head of service and supply with the additional task of managing staff positions. Hes the same rank as other heads but steps up as captain if the captain goes missing

**Comment by Cactus2388** [comment]
bleh, fixed my wording

**Comment by Ralta** [comment]
Remove Cargonia from HoP's purview? Sure.
Give QM Bridge access et al? Plsgodno

In all seriousness I'm not really sure why this is necessary. The QM functions just fine as they are; giving them additional access/responsibility seems kinda...redundant?

**Comment by FlattestGuitar** [comment]
Okay, so we've talked about this at length and couldn't really come up with really good reasons *not* to let it pass.

We're going to merge this and come back to the matter in a week or two, depending on how badly this flops.

Hail QM.

**Comment by ike709** [comment]
Glory to Cargonia.

**Comment by SamHPurp** [comment]
Independence may be revoked in future, but lets see how this goes.

**Comment by jakeramsay007** [comment]
>makes civilian jobs like janitor report to the Quartermaster, not the HoP
I think that may be an error or an oversight.

**Comment by ike709** [comment]
@jakeramsay007 that is why you never use find/replace at1am

**Pull request merged**